modules/notification/base/notifier.go

```diff
@@ -9,6 +9,8 @@ import (
 	"code.gitea.io/gitea/modules/repository"
 )
 
+//go:generate go run -mod=vendor main.go
```

Comment:
Yeah or just make build as we run go generate in build now. 

notifier.go needs a comment to this effect though.

Reply:
Added a few comments

Reply:
Why we need to generate that two files? Any advantage the new method than before?

Reply:
1. If you change notifier.go they will be both be kept up to date.
2. It would be beyond tiresome to write the notifier queue by hand and incredibly difficult to keep up to date.